thunder/clang/__init__.py

```diff
@@ -1602,6 +1602,16 @@ def floor(a: TensorLike | Number) -> TensorLike | Number:
     )
 
 
+@clangop()
+def frexp(a: TensorLike, *, out=None) -> tuple[TensorLike, TensorLike]:
```

Comment:
Do clang ops typically have an `out` argument?

Reply:
Nit: `frexp(a: TensorLike, /) -> ...`

Reply:
> Do clang ops typically have an out argument?

Yes, a few ops like `topk` and `sort`.

https://github.com/Lightning-AI/lightning-thunder/blob/a8c063676cbecad7c196dcfe4fbeacf85775c9c5/thunder/clang/__init__.py#L2165-L2184

Reply:
Like with the prims, we should probably not support `out` on clang ops